.github/workflows/stale.yml

```diff
@@ -21,7 +21,21 @@ jobs:
     - uses: actions/stale@v9
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'Stale issue message'
-        stale-pr-message: 'Stale pull request message'
         stale-issue-label: 'no-issue-activity'
         stale-pr-label: 'no-pr-activity'
+        exempt-issue-labels: Not Stale
+        exempt-pr-labels: Not Stale
+        stale-issue-message: >

```

Comment:
I don't think we should close stale issues, because it's more of a bandwidth issue than "obsolescence" per se (i.e., majority of current > 60 days are legitimate features pending implementation, but simply haven't been yet). For the most part, I've just been managing this manually with periodic sweep-throughs, but personally would rather avoid potential redundancy with stuff getting opened as "new" issues only because the same thing was auto-closed previously, etc.

Reply:
CC: @tran-christian